tests/network/vmi_multus.go

```diff
@@ -333,57 +346,64 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 			})
 		})
 
-		Context("VirtualMachineInstance with Linux bridge plugin interface", func() {
-			It("[test_id:1577]should create two virtual machines with one interface", func() {
-				By("checking virtual machine instance can ping the secondary virtual machine instance using Linux bridge CNI plugin")
-				interfaces := []v1.Interface{linuxBridgeInterface}
-				networks := []v1.Network{linuxBridgeNetwork}
-
-				vmiOne := createVMIOnNode(interfaces, networks)
-				vmiTwo := createVMIOnNode(interfaces, networks)
+		Context("With two virtual machines", func() {
+			var (
+				interfaces []v1.Interface
+				networks   []v1.Network
+				vmiOne     *v1.VirtualMachineInstance
+				vmiTwo     *v1.VirtualMachineInstance
+			)
+			JustBeforeEach(func() {
```

Comment:
Why not using `BeforeEach`?

Reply:
This change is not included anymore since the code has changed and the test is now parametrized with DescribeTable.
Generally though, JustBeforeEach is really nice construct that allows separation of parametrization  and resource creation. See [doc section](https://onsi.github.io/ginkgo/#separating-creation-and-configuration-justbeforeeach) for an example.

Reply:
Thanks, it better now